charts/newrelic-infrastructure/values.yaml

```diff
@@ -90,24 +86,11 @@ ksm:
   # Advanced users only. Setting this to `false` is not supported and will break the New Relic experience.
   enabled: true
   annotations: {}
-  tolerations:
-    - operator: "Exists"
-      effect: "NoSchedule"
-    - operator: "Exists"
-      effect: "NoExecute"
+  tolerations: []
   nodeSelector: {}
-  affinity:
-    podAffinity:
-      preferredDuringSchedulingIgnoredDuringExecution:
-        - podAffinityTerm:
-            topologyKey: kubernetes.io/hostname
-            labelSelector:
-              matchLabels:
-                app.kubernetes.io/name: kube-state-metrics
-          weight: 100
```

Comment:
As above: I do not like this change, values should be explicit unless we really really need them not to be.